### PR TITLE
Feat/868: Use global synthesizer config  properties for say verb

### PR DESCRIPTION
**Pull request by rammohan-y**
https://github.com/jambonz/jambonz-feature-server/issues/868

Use the global synthenizer config from config verb for say verb

### CODE REVIEW

File: lib/tasks/say.js [hunk 1]
```diff
@@ -210,7 +211,22 @@ class TaskSay extends TtsTask {
 
     await super.exec(cs);
     this.ep = ep;
+    if (cs.synthesizer) {
```
Comment (davehorton) [inline lib/tasks/say.js:214]: shouldnt this be done in the superclass as well?

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/tasks/say.js (hunk 1 above)
Comment (xquanluu) [reply lib/tasks/say.js:214]: Hi @davehorton  @rammohan-y 

I see that _synthesizeWithSpecificVendor is implemented in 2 places tts-task and say task and they are doing exactly the same excep som logging and notification that tts-task doing better.

I would suggest to migrate the _synthesizeWithSpecificVendor in say.js to tts-task and we have only one place to handle thing.

Comment (davehorton) [reply lib/tasks/say.js]: @rammohan-y I agree with @xquanluu's point above, can you make that change?

Comment (rammohan-y) [reply lib/tasks/say.js]: @davehorton moved _synthesizeWithSpecificVendor method from say.js to tts-task.js

Comment (rammohan-y) [reply lib/tasks/say.js]: > Thanks, please do a thorough regression test, as this is a big change

The following were testing for both say and gather  + say

config contains synthesizer config, say verb doesn't contain synthesizer: synthesizer config from config is taken
config does not contain sythesizer config, verb contains synthesizer: synthesizer config from verb is taken
both config and verb has synthesizer options: synthesizer config from verb is taken
both config and verb does not contain synthesizer options: system took what is configured in the application

Comment (rammohan-y) [reply lib/tasks/say.js]: Tested for playht, microsoft
Tested with disableTtsCache true

**Comment by davehorton** [comment]
can rebase @rammohan-y

**Comment by rammohanyadavalli** [comment]
> can rebase @rammohan-y

Yeah, rebased now @davehorton

**Review by davehorton — APPROVED** [review]

**Pull request merged**